packages/dict/src/dict.ts

```diff
@@ -0,0 +1,54 @@
+import { Predicate, Guard, Reducer, UnaryFn } from '@proem/function'
+
+export interface Dictionary {
+  [key:string]: any
+}
+
+export const map = <A extends Dictionary, B>(dict: A, mapfn: ([k, v]: [string, any]) => B): B[] => {
```

Comment:
Could use `UnaryFn<[string, any], B>` here to match the signature of `map.partial` below.